service-registry/src/main/java/org/apache/servicecomb/serviceregistry/client/http/AbstractClientPool.java

```diff
@@ -42,27 +45,41 @@
 
   public static final String PROXY_KEY = "sc.consumer";
 
+  private static final String EVENT_LOOP_POOL_SIZE = "servicecomb.service.registry.client.eventLoopPoolSize";
+
+  private static final String WORKER_POOL_SIZE = "servicecomb.service.registry.client.workerPoolSize";
```

Comment:
`servicecomb.service.registry.client.workerPoolSize` is used as the key of another configurations. This is a change not compatible. Could you update the corresponding docs?